docs/content/en/docs/concepts/eksapackages.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "EKS Anywhere curated packages"
+linkTitle: "EKS Anywhere curated packages"
+weight: 4
+date: 2022-04-12
+description: >
+  Explanation of the process of managing curated packages in an EKS Anywhere cluster
+---
+
+EKS Anywhere Curated Packages is a management system for installation, configuration and maintenance of additional components for your Kubernetes applications. Examples of these components may include Container Registry, Ingress, and LoadBalancer, etc.
```

Comment:
```suggestion
EKS Anywhere Curated Packages is a management system for installation, configuration and maintenance of additional components for your Kubernetes applications. Examples of these components may include Container Registry, Ingress, LoadBalancer, etc.
```

Reply:
I also think `for your Kubernetes applications` is a bit awkward, but assume someone requested that. Seems like `for your Kubernetes cluster` would be better